tests/test_objectives.py

```diff
@@ -23,15 +23,15 @@ def build_data(num_datapoints: int, num_dims: int, key, binary: bool):
             0.5
             * jnp.sign(
                 jnp.cos(
-                    3 * x[:, 1].reshape(-1, 1)
+                    3 * x[:, 0].reshape(-1, 1)
```

Comment:
drive-by fix: the previous code shouldn't have worked for num_dims=1 ... :thinking: